src/projects.js

```diff
@@ -0,0 +1,71 @@
+const projects = (() => {
+  const projectList = {
+    list: [],
+    currentProjectId: null,
+    getCurrent() {
+      return this.list[this.currentProjectId];
+    },
+  };
+
+  const project = (title, id) => {
+    const todoArr = [];
+    const getTitle = () => title;
+    const getId = () => id;
+
+    const setTitle = (newTitle) => {
+      title = newTitle;
+    };
+
+    const setId = (newId) => {
+      id = newId;
+    };
+
+    const getTodo = () => todoArr;
+
+    const appendTodo = (todo) => {
+      todoArr.push(todo);
+    };
+
+    return {
+      getTitle,
+      getId,
+      setTitle,
+      setId,
+      getTodo,
+      appendTodo,
+    };
+  };
+
+  const createProject = (newProjectTitle, newProjectId) => {
+    const newProject = project(newProjectTitle, newProjectId);
+    projectList[newProjectId] = {
+      projectTitle: newProjectTitle,
+      arrayOfToDos: [],
+    };
+
+    projectList.list.push(newProject);
+    // localStorage.setItem('projectList', JSON.stringify(projectList));
+    // console.log(JSON.parse(localStorage.getItem('projectList')));
```

Comment:
**REQUIRED**
Let's get rid of the commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from Pathwright)